AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -639,62 +646,93 @@ - (BOOL)isLayerBacked
   return _flags.layerBacked;
 }
 
-#pragma mark - Layout measurement calculation
+#pragma mark - Layout measurement and sizing
+
+- (ASSize)size
+{
+  ASDN::MutexLocker l(__instanceLock__);
+  return _size;
+}
+
+- (void)setSize:(ASSize)size
+{
+  ASDN::MutexLocker l(__instanceLock__);
+  if (ASSizeEqualToSize(_size, size) == NO) {
+    _size = size;
+    [self invalidateCalculatedLayout];
+  }
+}
+
+ASLayoutableSizeForwarding
+ASLayoutableSizeHelperForwarding
 
+// Deprecated
 - (CGSize)measure:(CGSize)constrainedSize
 {
-  return [self measureWithSizeRange:ASSizeRangeMake(CGSizeZero, constrainedSize)].size;
+    return [self layoutThatFits:ASSizeRangeMake(CGSizeZero, constrainedSize)].size;
+}
+
+- (ASLayout *)layoutThatFits:(ASSizeRange)constrainedSize
+{
+#pragma clang diagnostic push
```

Comment:
Better to call directly to `layoutThatFits:parentSize:` instead of into a deprecated method.